### PR TITLE
CDAP-4752 Add missing settings to Ambari installation docs

### PR DESCRIPTION
**Pull request by awholegunch**
Passes [Quick Build](http://builds.cask.co/browse/CDAP-DOB100-3) (build 3)

Revises installation documentation (in particular, Ambari) to fix
https://issues.cask.co/browse/CDAP-4752.

- Adds a number of Hadoop Configurations needed to each of the installations
- Does so using a common file referenced in all the installations.
- Fixes minor formatting errors in configuration.txt (indentation)
- Fixes text errors in system-requirements.rst (corrects the shell prompt, adds "sudo")

Pages of Interest:

- [Cloudera](http://builds.cask.co/artifact/CDAP-DOB100/shared/build-3/Docs-HTML/3.3.1-SNAPSHOT/en/admin-manual/installation/cloudera.html)
- [Ambari](http://builds.cask.co/artifact/CDAP-DOB100/shared/build-3/Docs-HTML/3.3.1-SNAPSHOT/en/admin-manual/installation/ambari.html)
- [MapR](http://builds.cask.co/artifact/CDAP-DOB100/shared/build-3/Docs-HTML/3.3.1-SNAPSHOT/en/admin-manual/installation/mapr.html)
- [Packages](http://builds.cask.co/artifact/CDAP-DOB100/shared/build-3/Docs-HTML/3.3.1-SNAPSHOT/en/admin-manual/installation/packages.html)

### CODE REVIEW

File: cdap-docs/admin-manual/source/installation/ambari.rst [hunk 1]
```diff
@@ -45,6 +45,23 @@ Preparing the Cluster
...
+.. Hadoop Configuration
+.. --------------------
+.. include:: ../_includes/installation/hadoop-configuration.txt
+      
+You can make these changes during the configuration of your cluster `using Ambari 
+<http://docs.hortonworks.com/HDPDocuments/Ambari-2.2.0.0/bk_Installing_HDP_AMB/content/_customize_services.html>`__.
+
+.. highlight:: console
+
+HDFS Permissions
```
Comment (gokulavasan) [inline cdap-docs/admin-manual/source/installation/ambari.rst]: Won't ``HDFS Permissions`` section be also common across all installations?

**Comment by gokulavasan** [comment]
@awholegunch Finished review. One question.

File: cdap-docs/admin-manual/source/installation/ambari.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/ambari.rst]: Added as a common section across all distributions.

**Comment by gokulavasan** [comment]
LGTM :+1:

**Pull request merged**